packit_service/worker/helpers/build/babysit.py

```diff
@@ -55,6 +57,13 @@
 logger = logging.getLogger(__name__)
 
 
+def celery_run_async(signatures: list[signature]) -> None:
```

Comment:
```suggestion
def celery_run_async(signatures: list[Signature]) -> None:
```

[`signature`](https://github.com/celery/celery/blob/dfe241ecb2bb8f536ddab83b6e18a806029a227d/celery/canvas.py#L2365) is a function, you are not passing a list of functions.

Reply:
I wasn't sure, but I was expecting mypy to complain 🤔...

Reply:
Yes, it's weird that it didn't.